pkg/skel/skel_test.go

```diff
@@ -93,8 +93,9 @@ var _ = Describe("dispatching to the correct callback", func() {
 		err := dispatch.pluginMain(cmdAdd.Func, cmdDel.Func, versionInfo)
 		if isRequired {
 			Expect(err).To(Equal(&types.Error{
-				Code: 100,
-				Msg:  "required env variables missing",
+				CniVersion: version.Current(),
+				Code:       100,
+				Msg:        "required env variables missing",
```

Comment:
Could we also add a testcase that sends an old CNI spec version and ensure that the returned error reports that old version?